File: HFSGuardaDiretorioVS2022/HFSGuardaDiretorio_MFC/hfsguardadir/comum/DiretorioXML.cpp
#include "stdafx.h"
#include "DiretorioXML.h"

namespace hfsguardadir {

namespace comum {

DiretorioXML::DiretorioXML()
{
    this->diretorio = new Diretorio();
}

Diretorio* DiretorioXML::getDiretorio() {
    return diretorio;
}

void DiretorioXML::setDiretorio(Diretorio *diretorio) {
    this->diretorio = diretorio;
}

}
}

File: HFSGuardaDiretorio_Java/hfsguardadir/src/main/java/hfsguardadir/comum/DiretorioXML.java
package hfsguardadir.comum;

import hfsguardadir.objetos.Diretorio;

public class DiretorioXML {
    private Diretorio diretorio;

    public DiretorioXML() {
        this.diretorio = new Diretorio();
    }
    
    public Diretorio getDiretorio() {
        return diretorio;
    }

    public void setDiretorio(Diretorio diretorio) {
        this.diretorio = diretorio;
    }    
    
}
